src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -366,9 +341,54 @@ The following table shows the default retention settings for our [New Relic One
   </tbody>
 </table>
 
+## Adjust retention values [#adjust-retention]
+
+You can adjust your retention periods to better support your data needs and meet compliance requirements. You might reduce retention values to optimize query performance. Or, you might increase retention for some data types to support more granular analysis over longer periods of time.  
+
+The ability to edit data retention periods requires a specific [edition of New Relic](https://newrelic.com/pricing).
```

Comment:
is there a difference between "edition" and "tier" ? I think I've seen both words used but not clear if they are interchangeable or mean very specific things.